rwengine/src/audio/SoundManager.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef _RWENGINE_SOUNDMANAGER_HPP_
 #define _RWENGINE_SOUNDMANAGER_HPP_
 
+#include <algorithm>
```

Comment:
You should include only what you use in that file. So `algorithm` is better included in the cpp file.

Reply:
IIRC some small function in headers are using it.